### PR TITLE
Add OutgoingCreditTransfer project code

### PR DESCRIPTION
**Pull request by DivyaPremanantha**
## Purpose
Add the OutgoingCreditTransfer WSO2 Micro Integrator project that has the implementation of a sample BIAN OutgoingCreditTransfer user scenario

### CODE REVIEW

File:  Outgoing Credit Transfer/OutgoingCreditTransfer_CAR/target/OutgoingCreditTransfer_CAR_1.0.0-SNAPSHOT.car
Comment (lalaji) [inline  Outgoing Credit Transfer/OutgoingCreditTransfer_CAR/target/OutgoingCreditTransfer_CAR_1.0.0-SNAPSHOT.car:1]: remove the target folders and content

File:  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.meta/~CorrespondentBankOperationsConfig_inputSchema.json.xml
Comment (lalaji) [inline  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.meta/~CorrespondentBankOperationsConfig_inputSchema.json.xml:1]: Do we need files inside .meta folder?

File:  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.settings/org.eclipse.core.resources.prefs
Comment (lalaji) [inline  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.settings/org.eclipse.core.resources.prefs:1]: do we need this file?

File:  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.settings/.meta/~org.eclipse.core.resources.prefs.xml
Comment (lalaji) [inline  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.settings/.meta/~org.eclipse.core.resources.prefs.xml:1]: Do we need this? Can you further check,there are some files inside .meta/ .settings folders which don't need to commit.

File:  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.meta/~CorrespondentBankOperationsConfig_inputSchema.json.xml
Comment (DivyaPremanantha) [reply  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.meta/~CorrespondentBankOperationsConfig_inputSchema.json.xml:1]: Fixed in 7bfc9f6569282a2cdd6c73262e1bd69075de2c38

File:  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.settings/org.eclipse.core.resources.prefs
Comment (DivyaPremanantha) [reply  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.settings/org.eclipse.core.resources.prefs:1]: Fixed in 7bfc9f6569282a2cdd6c73262e1bd69075de2c38

File:  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.settings/.meta/~org.eclipse.core.resources.prefs.xml
Comment (DivyaPremanantha) [reply  Outgoing Credit Transfer/OutgoingCreditTransfer_REG/.settings/.meta/~org.eclipse.core.resources.prefs.xml:1]: Fixed in 7bfc9f6569282a2cdd6c73262e1bd69075de2c38

File:  Outgoing Credit Transfer/OutgoingCreditTransfer_CAR/target/OutgoingCreditTransfer_CAR_1.0.0-SNAPSHOT.car
Comment (DivyaPremanantha) [reply  Outgoing Credit Transfer/OutgoingCreditTransfer_CAR/target/OutgoingCreditTransfer_CAR_1.0.0-SNAPSHOT.car:1]: Removed in 60d84a2b456a1b17c49985b2559c29b479e3255a

**Review by lalaji — APPROVED** [review]

**Pull request merged**